.github/workflows/contracts-tests.yml

```diff
@@ -26,14 +26,14 @@ jobs:
           node-version: 18
 
       - name: 'Install the dependencies'
-        run: 'yarn install'
+        run: 'yarn install --frozen-lockfile'
 
       - name: 'Build the contracts'
         run: 'yarn build'
         env:
           INFURA_API_KEY: ${{ secrets.INFURA_API_KEY }}
 
       - name: 'Test the contracts and generate the coverage report'
-        run: 'yarn coverage >> $GITHUB_STEP_SUMMARY'
```

Comment:
Explainer: We don't pipe the output into `GITHUB_STEP_SUMMARY`. It's more straightforward to have them in the runner output instead of the summary. 3rd party devs likely won't understand what is happening here, so I removed this.